pokete_classes/fightmap/__init__.py

```diff
@@ -147,8 +171,8 @@ def add_2(self, player):
         time.sleep(SPEED_OF_TIME * 0.05)
         self.show()
         player.curr.ico.add(self, 3, self.height - 10)
-        self.box.add_c_obs([atc.label for atc in player.curr.attack_obs])
-        self.box.set_index(0)
+        self.box.box.add_c_obs([atc.label for atc in player.curr.attack_obs])
+        self.box.box.set_index(0)
```

Comment:
just a tiny nitpick, but `.box.box` seems weird and possibly prone to errors, especially without static type checking in python, is there a better factoring for this?